builder/oracle/classic/step_add_keys.go

```diff
@@ -0,0 +1,67 @@
+package classic
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-oracle-terraform/compute"
+	uuid "github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/packer/helper/multistep"
+	"github.com/hashicorp/packer/packer"
+)
+
+type stepAddKeysToAPI struct{}
+
+func (s *stepAddKeysToAPI) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
+	// get variables from state
+	ui := state.Get("ui").(packer.Ui)
+	ui.Say("Adding SSH keys to API...")
+	config := state.Get("config").(*Config)
+	client := state.Get("client").(*compute.ComputeClient)
+
+	// grab packer-generated key from statebag context.
+	sshPublicKey := strings.TrimSpace(state.Get("publicKey").(string))
+
+	// form API call to add key to compute cloud
+	uuid_string, err := uuid.GenerateUUID()
+	if err != nil {
+		ui.Error(fmt.Sprintf("Error creating unique SSH key name: %s", err.Error()))
+		return multistep.ActionHalt
+	}
+	sshKeyName := fmt.Sprintf("/Compute-%s/%s/packer_generated_key_%s",
```

Comment:
do we want to support the case where the user is bringing their own key? Gets more complicated (see `builder/amazon/common/step_key_pair.go`)...